feeluown/ui.py

```diff
@@ -41,15 +41,76 @@
 logger = logging.getLogger(__name__)
 
 
+class Button(QPushButton):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # 按钮文字一般是一个 symbol，长度控制为 40 是满足需求的
+        self.setMaximumWidth(40)
+
+
 class VolumeSlider(QSlider):
-    def __init__(self, parent=None):
+    def __init__(self, parent=None, initial_volume=100):
         super().__init__(parent)
 
         self.setOrientation(Qt.Horizontal)
-        #self.setRange(0, 100)   # player volume range
-        #self.setValue(100)
+        self.setMinimum(0)
+        self.setMaximum(100)
+        self.setTickInterval(1)
+        self.setValue(initial_volume)
         self.setToolTip('调教播放器音量')
 
+    def mouseReleaseEvent(self, event):
+        value = QStyle.sliderValueFromPosition(self.minimum(), self.maximum(),
+                                               event.x(), self.width())
+        self.setValue(value)
+
+
+class VolumeControler(QFrame):
```

Comment:
How about creating a `VolumeButton` component in `feeluown.components` package? The `ui` module seems more like a glue layer.